src/connecpy/errors.py

```diff
@@ -23,8 +21,12 @@ class Errors(Enum):
     Internal = "internal"
     Unavailable = "unavailable"
     DataLoss = "data_loss"
-    Malformed = "malformed"
+    Unauthenticated = "unauthenticated"
+
+    # Custom error codes not defined by Connect
```

Comment:
I think these should be removed, it's confusing to provide errors that connect doesn't define. I think `BadRoute` -> `Unimplemented` and `Malformed` -> `InvalidArgument`. Is it ok? I would do it in another PR.

Reply:
yeah, thank you, please do it.

Reply:
Thanks - I'll do it in another PR after merging this one since it will require some reworking too